references/cli/inferless-runtime.mdx

```diff
@@ -14,8 +14,7 @@ $ inferless runtime COMMAND [OPTIONS]  [ARGS]...
 
 - `generate`: use to generate a new runtime from your virtual env
 - `list`: list all runtimes.
-- `select [options]`: use an existing runtime with the current model
-- `upload [options]`: create or update runtime on inferless with yaml.
+- `create [options]`: create runtime on inferless with yaml.
 - `patch [options]`: update the runtime with the config file
```

Comment:
Create a new version of runtime with modified packages using yaml

Reply:
added